kedro/config/logging.yml

```diff
@@ -1,44 +1,21 @@
 version: 1
```

Comment:
Some of these changes are just formatting. What's actually changed here is:
* removed `info_file_handler` and `error_file_handler`. These are now only defined on the project-side logging config
* removed special case `kedro.framework.cli.hooks.manager` logger (which was the same as the default, just without the file-based handlers enabled)